thoth/storages/graph/postgres.py

```diff
@@ -6030,7 +6034,6 @@ def sync_solver_result(self, document: dict, *, force: bool = False) -> None:
                         os_version=ecosystem_solver.os_version,
```

Comment:
`package_metadata` needs to be provided here

Reply:
Oh, sorry. I misread

Reply:
no problem @KPostOffice, now I see what you meant, you were referring to the id of package_metadata. All good now :)